tsdb/mapper.go

```diff
@@ -313,7 +313,7 @@ func (lm *SelectMapper) nextChunkRaw() (interface{}, error) {
 		}
 		cursor := lm.cursors[lm.currCursorIndex]
 
-		k, v, t := cursor.Next(lm.queryTMin, lm.queryTMax, lm.selectFields, lm.whereFields)
+		k, v := cursor.Next(lm.queryTMin, lm.queryTMax, lm.selectFields, lm.whereFields)
```

Comment:
With the addition of `Tags()` to the iterator interface, we no longer needed to expand this signature and can ask for tags only when necessary.  This is an optimization and one that @otoolep was concerned about when we originally started returning tags with each `Next`.